README.md

```diff
@@ -58,19 +58,20 @@ After installing the tool you can run `pmd.sh` or `pmd.bat` similar to the follo
 
 - Next, configure (add) the ruleset from this repo by URL to always be up-to-date:
 
-      Settings > Other Settings > PMD > RuleSets > + > Choose RuleSet > drop down > Choose 'jpinpoint-rules' 
+      Settings > Tools > PMD > RuleSets > + > Choose RuleSet > drop down > Choose 'jpinpoint-rules' 
 
 - Alternatively, download and add your local copy: *rulesets/java/jpinpoint-rules.xml*
```

Comment:
can turn this into a link? As here:

[rulesets/java/jpinpoint-rules.xml](https://raw.githubusercontent.com/jborgers/PMD-jPinpoint-rules/refs/heads/pmd7/rulesets/java/jpinpoint-rules.xml)

```text
[rulesets/java/jpinpoint-rules.xml](https://raw.githubusercontent.com/jborgers/PMD-jPinpoint-rules/refs/heads/pmd7/rulesets/java/jpinpoint-rules.xml)
```

Reply:
Add this link also at Eclipse section